apps/api/migrations/changes-migration.ts

```diff
@@ -0,0 +1,210 @@
+import { AppModule } from '../src/app.module';
+import {
+  NotificationTemplateRepository,
+  MessageTemplateRepository,
+  NotificationGroupRepository,
+  EnvironmentRepository,
+  ChangeRepository,
+  MemberRepository,
+  OrganizationRepository,
+} from '@novu/dal';
+import { ChangeEntityTypeEnum, MemberRoleEnum } from '@novu/shared';
+import { CreateChange } from '../src/app/change/usecases/create-change.usecase';
+import { CreateChangeCommand } from '../src/app/change/usecases/create-change.command';
+import { CreateEnvironment } from '../src/app/environments/usecases/create-environment/create-environment.usecase';
+import { CreateEnvironmentCommand } from '../src/app/environments/usecases/create-environment/create-environment.command';
+import { ApplyChange } from '../src/app/change/usecases/apply-change/apply-change.usecase';
+import { ApplyChangeCommand } from '../src/app/change/usecases/apply-change/apply-change.command';
+
+export async function run(): Promise<void> {
+  console.log('Script started');
+  console.log('');
+  const app = await NestFactory.create(AppModule, {
+    logger: false,
+  });
+  console.log('');
+  console.log('App created');
+
+  const memberRepository = app.get(MemberRepository);
+  const organizationRepository = app.get(OrganizationRepository);
+  const notificationTemplateRepository = app.get(NotificationTemplateRepository);
+  const messageTemplateRepository = app.get(MessageTemplateRepository);
+  const notificationGroupRepository = app.get(NotificationGroupRepository);
+  const environmentRepository = app.get(EnvironmentRepository);
+  const createChangeUseCase = app.get(CreateChange);
+  const createEnvironment = app.get(CreateEnvironment);
+  const applyChange = app.get(ApplyChange);
+  console.log('Repositories and usecases created');
+
+  const orgs = await organizationRepository.find({});
+  console.log(`${orgs.length} Orgs found`);
+  for (const org of orgs) {
+    console.log(`Migrating org ${org._id}`);
+    const member = await memberRepository.findOne({
+      roles: MemberRoleEnum.ADMIN,
+      _organizationId: org._id,
+    });
+
+    console.log(`Using user ${member._id} to migrate org ${org._id}`);
+    console.log('');
+    const environments = await environmentRepository.findOrganizationEnvironments(org._id);
+    console.log(`Found ${environments.length} environments`);
+
+    if (environments.length === 2) {
+      console.log(`Connects Production environment to Development environment`);
+      const prod = environments.reduce((prev, current) => {
+        return current.name === 'Production' ? current : prev;
+      }, environments[1]);
+      const dev = environments.reduce((prev, current) => {
+        return current.name !== prod.name ? current : prev;
+      }, environments[0]);
+
+      prod._parentId = dev._id;
+
+      await environmentRepository.update(
+        {
+          _id: prod._id,
+        },
+        {
+          $set: prod,
```

Comment:
```suggestion
          $set: {
             _parentId: dev._id
          }
```

Would suggest only setting a specific field rather than overriding the entire object, not critical it would still work tho.